### PR TITLE
fix(alert-rules): disable clickhouse/prompql for anomaly detection

### LINKED ISSUE #11022: Anomaly alert preview bug with Click House query
## Bug description

 I'm having a problem when building an anomaly alert with ClickHouse query  using distributed_signoz_index_v3.

The query is working on threshold alert but crashes when using it in an anomaly alert:

This is the ClickHouse query
```
SELECT
      toStartOfMinute(trace_start) AS ts,
      max(trace_duration_seconds) AS value
  FROM (
      SELECT
          traceID,
          (max(toUnixTimestamp64Nano(timestamp) + durationNano) - min(toUnixTimestamp64Nano(timestamp))) / 1e9
  AS trace_duration_seconds,
          min(timestamp) AS trace_start
      FROM signoz_traces.distributed_signoz_index_v3
      WHERE timestamp BETWEEN {{.start_datetime}} AND dateAdd(hour, 6, {{.end_datetime}})
      GROUP BY traceID
  )
  GROUP BY ts
  ORDER BY ts ASC
```

Basically I'm getting the duration of traces taking into account their spans

The query range GET seems working fine because it returns correct data (aggregations populated, 2 months of trace history available):
```
{
   "aggregations": [{
      "alias": "__result_0",
      "series": [{ "values": [...] }]
    }]
  }
```
  But anomalyDetectionData in the component always has empty predictions:
```
 anomalyDetectionData[0] = {
    series: [{…}],         
    predictedSeries: [],   
    upperBoundSeries: [],  
    lowerBoundSeries: [],  
  }
```
  This causes a React crash in AnomalyAlertEvaluationView:

```
  TypeError: Cannot destructure property 'values' of 'a[r]' as it is undefined
      at N (getUplotChartData.ts:152:20)
      at se (getUplotChartData.ts:190:9)
      at AnomalyAlertEvaluationView.tsx:74:21
```

## Expected behavior

The anomaly alert preview should be working. 

## How to reproduce

1. Create a new anomaly alert with this query and try to preview it via hitting the button `Run query`:

This is the query:

```
SELECT
      toStartOfMinute(trace_start) AS ts,
      max(trace_duration_seconds) AS value
  FROM (
      SELECT
          traceID,
          (max(toUnixTimestamp64Nano(timestamp) + durationNano) - min(toUnixTimestamp64Nano(timestamp))) / 1e9
  AS trace_duration_seconds,
          min(timestamp) AS trace_start
      FROM signoz_traces.distributed_signoz_index_v3
      WHERE timestamp BETWEEN {{.start_datetime}} AND dateAdd(hour, 6, {{.end_datetime}})
      GROUP BY traceID
  )
  GROUP BY ts
  ORDER BY ts ASC

```

## Version information 
* **Signoz version**: Cloud v0.119.0
* **Browser version**: Brave 1.89.132 (Official Build) (arm64)](https://brave.com/latest/)
Chromium: 147.0.7727.56
* **Your OS and version**:  macOS 15.6.1 (24G90)
* **Your CPU Architecture**(ARM/Intel): ARM

## Some screenshots
Image
Image

### PR DESCRIPTION
**Pull request by H4ad**
## Pull Request

---

Today the Anomaly Detection does not support PromQL and Clickhouse Queries, this PR disable the UI to show them.

Before:

https://github.com/user-attachments/assets/28f43374-3e61-4c4e-adc8-6e21406e3e6c

After:

https://github.com/user-attachments/assets/cece8a9d-2d1b-4884-973b-32fe4c4a8896

Closes #11022 since it's not supported, created #11034 and #11036 to track the support.

---

### ✅ Change Type
_Select all that apply_

- [ ] ✨ Feature
- [x] 🐛 Bug fix
- [ ] ♻️ Refactor
- [ ] 🛠️ Infra / Tooling
- [ ] 🧪 Test-only

---

The needed data to render the anomaly was not sent by the backend due to not being supported.

Code expecting fields that was not sent.

Disable it and ensure the panels are not rendered.

---

- Tests added/updated:
- Manual verification:
- Edge cases covered:

---

- Blast radius: Alerts
- Potential regressions: None
- Rollback plan: Revert this comment.

---

| Field | Value |
|------|-------|
| Deployment Type | Cloud / OSS / Enterprise |
| Change Type | Bug Fix |
| Description | The Clickhouse Query and PromQL was enabled for Alerts Anomaly Detection when they were not supported yet. |

---

### 📋 Checklist
- [ ] Tests added or explicitly not required
- [x] Manually tested
- [ ] Breaking changes documented
- [ ] Backward compatibility considered

### CODE REVIEW

**Review by AshwinBhatkal — DISMISSED** [review]

File: frontend/src/lib/uPlotLib/utils/getUplotChartData.ts [hunk 1]
```diff
@@ -162,10 +167,10 @@ const processAnomalyDetectionData = (
 
 			for (let i = 0; i < length; i++) {
 				timestamps[i] = seriesValues[i].timestamp / 1000;
-				values[i] = seriesValues[i].value;
-				predicted[i] = predictedValues[i].value;
-				upperBound[i] = upperBoundValues[i].value;
-				lowerBound[i] = lowerBoundValues[i].value;
+				values[i] = seriesValues[i]?.value ?? null;
```
Comment (AshwinBhatkal) [inline frontend/src/lib/uPlotLib/utils/getUplotChartData.ts:170]: Is there a better way to handle this. Looks like patch work for now

Comment (H4ad) [reply frontend/src/lib/uPlotLib/utils/getUplotChartData.ts:170]: I didn't want to change much of the behavior of this function since I don't know how much this will affect other areas, so this is the safest patch I could apply since this code should not execute with invalid data (ideally).

**Review by AshwinBhatkal — APPROVED** [review]
